tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -297,6 +304,41 @@ def compute_stats(stats):
     print("\nTotal number of replaced kernel launches: {0:d}".format(len(stats["kernel_launches"])))
 
 
+def add_dim3(kernel_string, cuda_kernel):
```

Comment:
You can instead modify processKernelLaunches and directly add the dim3's. Though this is fine for now. I can refactor everything in the future.